nextflow.config

```diff
@@ -0,0 +1,59 @@
+params {
```

Comment:
Shouldn't all of this config live in the milton module?

Reply:
because of the last issue I had with singularity I copied all here.
We should create a PR to the milton profile then, or how can we resolve it?